unwrappr.gemspec

```diff
@@ -52,6 +52,7 @@ Gem::Specification.new do |spec| # rubocop:disable Metrics/BlockLength:
   spec.add_development_dependency 'rubocop', '>= 0.49.0'
 
   spec.metadata = {
+    'rubygems_mfa_required' => 'true',
```

Comment:
Maybe continue to order attributes alphabetically?

Reply:
👍 Updated in [afb566c](https://github.com/envato/unwrappr/pull/92/commits/afb566c298752f66d5a7d1d7d80a69fd7d5c5fdb)